Day3/ronejfourn/assignment2/free_list.c

```diff
@@ -0,0 +1,64 @@
+#include "os_virt_mem.h"
+
+#define MIN_VIRT_SIZE PAGE_SIZE
+#define MIN_CHUNK_SIZE 256
+// arbitrary maximum no. of mappings
+#define MAXM_MAP 4096
+
+typedef struct chunk {
+    struct chunk *next;
+} chunk;
```

Comment:
i think this will be easier if you do smth like ```c
typedef struct chunk {
   struct chunk *next;
   char mem[MIN_CHUNK_SIZE - sizeof(next)];
};
```